doc/us-en/setup/Build-from-Source.md

```diff
@@ -20,57 +20,31 @@ go env  # View Go's environment variables
 go version # View Go's version
 ```
 
-_**NOTE**: OpenEdge requires that the compiled version of Go be above 1.10.0._
+**NOTE**: OpenEdge requires that the compiled version of Go should be above 1.10.0.
 
-#### Install Docker
+#### Install the container runtime
 
-- The official Dockerfile is offered for multi-stage builds. If you need to build the relevant image yourself, The version of docker you installed should be above 17.05;
-- According to the [Official Release Log](https://docs.docker.com/engine/release-notes/#18092), the version of docker lower than 18.09.2 has some security implications. It is recommended to install/update the docker to 18.09.2 and above.
-
-Can be installed by the following command(Suitable for linux-like systems, [Supported Platforms](./Support-platforms.md)):
+In **docker** container mode, OpenEdge relies on docker container. If `docker` is not installed yet, users can install the latest version of docker (for Linux-like systems) with the following command:
 
 ```shell
 curl -sSL https://get.docker.com | sh
 ```
 
-**Ubuntu**
-
-Command:
-
-```shell
-sudo snap install docker # Ubuntu16.04 after
-sudo apt-get install docker.io # Ubuntu 16.04 before
-```
-
-**CentOS**
-
-Command:
-
-```shell
-yum install docker
-```
-
-**NOTE**: After the docker installation is complete, use the following command to view the installed version of docker.
+After the installation is complete, users can check the version of `docker`:
```

Comment:
Once the installation is complete, users can check the version of `docker`:

Reply:
View the version of installed docker: